google/resource_compute_region_instance_group_manager.go

```diff
@@ -227,6 +245,7 @@ func getManager(d *schema.ResourceData, meta interface{}) (*computeBeta.Instance
 
 	if err != nil {
 		handleNotFoundError(err, d, fmt.Sprintf("Region Instance Manager %q", d.Get("name").(string)))
+		return nil, err
```

Comment:
Good catch! This should just be `return handleNotFoundError(...)`

Reply:
So, the signature doesn't actually support that specific code snippet.

What's curious, though, is that we'd be returning `nil, nil` in the case of a not found error if we used the output of `handleNotFoundError`.  This feels totally wrong, but I'm willing to deal if that's simply the way things shook out in the provider.

Reply:
So what we want overall is to make sure that if the IGM isn't found when we try to read it, that we remove it from state. This is a totally valid case and shouldn't cause Terraform to error. So we don't want to return the actual error here, because then a not found will cause our terraform command to fail. Here's how we handle it for `resource_compute_instance`:
https://github.com/terraform-providers/terraform-provider-google/blob/ab385ff5740b7a53ca6302d6c6214c273e178aa0/google/resource_compute_instance.go#L773-L776
https://github.com/terraform-providers/terraform-provider-google/blob/ab385ff5740b7a53ca6302d6c6214c273e178aa0/google/resource_compute_instance.go#L588-L603

Reply:
Makes sense.  I'll take care of it.